src/adapters/MeetingsAdapter.js

```diff
@@ -0,0 +1,90 @@
+import {throwError} from 'rxjs';
+
+import WebexAdapter from './WebexAdapter';
+
+/**
+ * This is a base class that defines the interface that maps meetings data.
+ * Developers that want to extend `MeetingsAdapter` must implement all of its methods,
+ * adhering to the exact parameters and structure of the returned objects.
+ */
+export default class MeetingsAdapter extends WebexAdapter {
+  /**
+   * A meeting object that allows users to have a WebRTC meeting.
+   *
+   * @typedef {Object} Meeting
+   * @property {string}  ID The meeting identifier.
+   * @property {string}  meetingTitle The title of the meeting.
+   * @property {Date}    meetingTimeStart The time and date of the start of the meeting.
+   * @property {Date}    meetingTimeEnd The time and date of the end of the meeting.
```

Comment:
Is there a concept of meeting "duration"? i would rather have that information and calculate the end time.

Reply:
I was trying to reduce the amount of work required on the front end since this will just be converted to local timezone and rendered.